src/utils/text/expressions.js

```diff
@@ -0,0 +1,29 @@
+import { removePunctuation, tokenize } from "./preprocessing";
+
+function isExpression(text) {
+  return text.includes(" ");
+}
+
+function isTextInSentence(text, sentence) {
+  if (isExpression(text)) {
+    return sentence.includes(text);
+  }
+  let tokens = tokenize(sentence);
+  tokens = tokens.map((each) => removePunctuation(each));
+  return tokens.includes(removePunctuation(text));
+}
+
+function isWordIncluded(input, expression) {
+  let isWordIncluded = false;
+  let wordsInAnswer = expression.split(" ");
+  let wordsInInput = input.split(" ");
+  if (wordsInInput.length === 1)
+    wordsInAnswer.forEach((word) => {
+      if (input === word) isWordIncluded = true;
+    });
+  return isWordIncluded;
+}
+
+export { isWordIncluded };
```

Comment:
`export { isWordIncluded, isTextInSentence, isExpression};` is the way we usually export multiple functions?

Reply:
yes, that was the refactoring engine of WebStorm that did that. I looked at it, it looked strange, and then thought... whatever :) but let's do it our way!